Kitura: appsody test failure

Appsody stack validate fails with this during the test stage:

[Container] Running command:  swift test
[Container] [ControllerDebug] New process created with pid 16
[Container] [ControllerDebug] Started RUN/DEBUG/TEST process
/project/user-app/Tests/ApplicationTests/KituraStackTests.swift:121:19: error: use of undeclared type 'URLRequest'
[Container] private extension URLRequest {
[Container]                   ^~~~~~~~~~
[Container] /project/user-app/Tests/ApplicationTests/KituraStackTests.swift:44:9: error: use of unresolved identifier 'URLRequest'
[Container]         URLRequest(forTestWithMethod: "GET", port: AppsodyKitura.port, route: AppsodyKitura.livenessPath)?
[Container]         ^~~~~~~~~~
[Container] /project/user-app/Tests/ApplicationTests/KituraStackTests.swift:65:9: error: use of unresolved identifier 'URLRequest'
[Container]         URLRequest(forTestWithMethod: "GET", port: AppsodyKitura.port, route: AppsodyKitura.openAPIPath)?
[Container]         ^~~~~~~~~~
[Container] /project/user-app/Tests/ApplicationTests/KituraStackTests.swift:101:9: error: use of unresolved identifier 'URLRequest'
[Container]         URLRequest(forTestWithMethod: "GET", port: AppsodyKitura.port, route: AppsodyKitura.metricsPath)?
[Container]         ^~~~~~~~~~
[Container] /project/user-app/Tests/ApplicationTests/KituraStackTests.swift:131:13: error: use of unresolved identifier 'addValue'
[Container]             addValue("application/json", forHTTPHeaderField: "Content-Type")
[Container]             ^~~~~~~~
[Container] /project/user-app/Tests/ApplicationTests/KituraStackTests.swift:132:13: error: use of unresolved identifier 'httpMethod'
[Container]             httpMethod = method
[Container]             ^~~~~~~~~~
[Container] /project/user-app/Tests/ApplicationTests/KituraStackTests.swift:133:13: error: use of unresolved identifier 'cachePolicy'
[Container]             cachePolicy = .reloadIgnoringCacheData
[Container]             ^~~~~~~~~~~
[Container] /project/user-app/Tests/ApplicationTests/KituraStackTests.swift:135:17: error: use of unresolved identifier 'httpBody'
[Container]                 httpBody = body
[Container]                 ^~~~~~~~
[Container] /project/user-app/Tests/ApplicationTests/KituraStackTests.swift:148:25: error: use of unresolved identifier 'url'
[Container]         guard let url = url else {
[Container]                         ^~~
[Container] /project/user-app/Tests/ApplicationTests/KituraStackTests.swift:151:28: error: use of unresolved identifier 'httpMethod'
[Container]         guard let method = httpMethod, let headers = allHTTPHeaderFields, let host = url.host, let port = url.port else {
[Container]                            ^~~~~~~~~~
[Container] /project/user-app/Tests/ApplicationTests/KituraStackTests.swift:151:54: error: use of unresolved identifier 'allHTTPHeaderFields'
[Container]         guard let method = httpMethod, let headers = allHTTPHeaderFields, let host = url.host, let port = url.port else {
[Container]                                                      ^~~~~~~~~~~~~~~~~~~
[Container] /project/user-app/Tests/ApplicationTests/KituraStackTests.swift:188:27: error: use of unresolved identifier 'httpBody'
[Container]         if let dataBody = httpBody {
[Container]                           ^~~~~~~~
[260/264] W[Debug] CLI exit error is:  exit status 1

--- a/incubator/kitura/stack.yaml
+++ b/incubator/kitura/stack.yaml
@@ -1,5 +1,5 @@
 name: Kitura
-version: 0.2.2
+version: 0.2.3
 description: Runtime for Kitura applications
 license: Apache-2.0
 language: swift

--- a/incubator/kitura/templates/default/Tests/ApplicationTests/KituraStackTests.swift
+++ b/incubator/kitura/templates/default/Tests/ApplicationTests/KituraStackTests.swift
@@ -3,11 +3,7 @@ import Kitura
 import KituraNet
 import XCTest
 import AppsodyKitura
-#if swift(>=4.1)
-   #if canImport(FoundationNetworking)
-     import FoundationNetworking
-   #endif
- #endif
+import FoundationNetworking
 
 @testable import Application
 

--- a/incubator/kitura/templates/default/Tests/ApplicationTests/KituraStackTests.swift
+++ b/incubator/kitura/templates/default/Tests/ApplicationTests/KituraStackTests.swift
@@ -7,7 +7,7 @@ import AppsodyKitura
    #if canImport(FoundationNetworking)
      import FoundationNetworking
    #endif
- #endif
+#endif
 
 @testable import Application
 

--- a/incubator/kitura/templates/default/Tests/ApplicationTests/KituraStackTests.swift
+++ b/incubator/kitura/templates/default/Tests/ApplicationTests/KituraStackTests.swift
@@ -3,7 +3,11 @@ import Kitura
 import KituraNet
 import XCTest
 import AppsodyKitura
-import FoundationNetworking
+#if swift(>=4.1)
+   #if canImport(FoundationNetworking)
+     import FoundationNetworking
+   #endif
+ #endif
 
 @testable import Application
 

--- a/incubator/kitura/templates/default/Tests/ApplicationTests/KituraStackTests.swift
+++ b/incubator/kitura/templates/default/Tests/ApplicationTests/KituraStackTests.swift
@@ -3,6 +3,11 @@ import Kitura
 import KituraNet
 import XCTest
 import AppsodyKitura
+#if swift(>=4.1)
+   #if canImport(FoundationNetworking)
+     import FoundationNetworking
+   #endif
+ #endif
 
 @testable import Application
 

--- a/incubator/kitura/templates/default/Tests/ApplicationTests/KituraStackTests.swift
+++ b/incubator/kitura/templates/default/Tests/ApplicationTests/KituraStackTests.swift
@@ -3,11 +3,10 @@ import Kitura
 import KituraNet
 import XCTest
 import AppsodyKitura
-#if swift(>=4.1)
-   #if canImport(FoundationNetworking)
-     import FoundationNetworking
-   #endif
+#if canImport(FoundationNetworking)
+   import FoundationNetworking
 #endif
+
 
 @testable import Application
 

